src/inference/src/dev/core_impl.hpp

```diff
@@ -117,6 +138,19 @@ class CoreImpl : public InferenceEngine::ICore, public std::enable_shared_from_t
         mutable std::mutex _cacheConfigMutex;
         CacheConfig _cacheConfig;
         std::map<std::string, CacheConfig> _cacheConfigPerDevice;
+
+        mutable std::mutex _core_property_mutex;
```

Comment:
Why do we need to have a new mutex?

Reply:
Maybe we can merge the 2 mutexes together, Will try do it.